File: clover/src/app/entities/flightRate/flight-rate.ts
import { NumberFilterParam } from '../number-filter-param/number-filter-param';

export class FlightRate {

    rateIdd:number;
    userIdd:number;
    flightInfo2FlightID:number;
    ocena:number;
    companyIdd:number;

    constructor(   rateIdd:number,userIdd:number,flightIdd:number,ocena:number,companyIdd:number){
        this.rateIdd=rateIdd;
        this.userIdd=userIdd;
        this.flightInfo2FlightID=flightIdd;
        this.ocena=ocena;
        this.companyIdd=companyIdd;
    }
}

File: Backend/AirlineAPI/Models/flightRate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineAPI.Models
{
    public class flightRate
    {
        [Key]
        public int RateIdd { get; set; }

        [Required]
        public int UserIdd { get; set; }
        [Required]

        public int FlightInfo2FlightID { get; set; }
        [Required]

        public int Ocena { get; set; }
        [Required]
        public int CompanyIdd { get; set; }


    }
}
